src/api/lib/activexml/transport.rb

```diff
@@ -437,5 +437,11 @@ def handle_response(http_response)
       message = http_response.to_s if message.blank?
       raise Error, message.force_encoding('UTF-8')
     end
+
+    private
+
+    def symbolified_model(model_class)
```

Comment:
[Nitpick] `symbolified_model` => `symbolized_model`

Reply:
Nitpick²: How about symbolize_model?

Reply:
I thought it was strange, but as it was like that everywhere 🤣